polyply/tests/test_gen_seq.py

```diff
@@ -99,76 +111,81 @@ def test_find_termini():
     for node in [0, 2, 4, 5]:
         assert node in termini
 
+
 def test_annote_modifications():
     modf = ["1:NH2", "2:NH3"]
     graph = nx.Graph()
     graph.add_edges_from([(0, 1), (1, 2), (1, 3), (3, 4), (3, 5)])
-    nx.set_node_attributes(graph, {0:1, 1:1, 2:1, 3:2, 4:2, 5:2}, "seqid")
-    nx.set_node_attributes(graph, {0:"PPI", 1:"PPI", 2:"PPI", 3:"PPI", 4:"PPI", 5:"PPI"}, "resname")
-    _apply_teminii_modifications(graph, modf)
+    nx.set_node_attributes(
+        graph, {0: 1, 1: 1, 2: 1, 3: 2, 4: 2, 5: 2}, "seqid")
+    nx.set_node_attributes(
+        graph, {0: "PPI", 1: "PPI", 2: "PPI", 3: "PPI", 4: "PPI", 5: "PPI"}, "resname")
```

Comment:
Definitely nitpicking here. Feel free to ignore.

I'm not super happy with this type of linebreaks (also earlier). I'd rather keep it on one line, or if you have to break it:
```
something_or_other(
    ...,
    ...,
)
```
So closing bracket also on newline.